lib/class-wp-theme-json-gutenberg.php

```diff
@@ -185,6 +190,13 @@ class WP_Theme_JSON_Gutenberg {
 				),
 			),
 		),
+		array(
+			'path'          => array( 'color', 'duotone' ),
+			'value_func'    => 'gutenberg_get_duotone_filter_property',
+			'value_args'    => array( 'slug', 'colors' ),
+			'css_var_infix' => 'duotone-filter',
```

Comment:
What do you think removing the `-filter` and making this var `--wp--preset--duotone--{value}` instead? The rest follow that pattern: `--wp--preset--color--{value}`, `--wp--preset--font-size--{value}`, etc. Not opinionated, just think that it's shorter without losing the meaning.